docs/channel.md

```diff
@@ -424,6 +424,47 @@ Y
 
 See also: [fromList](#fromlist) factory method.
 
+(channel-topic)=
+
+### topic
+
+:::{versionadded} 23.11.0-edge
+:::
+
+:::{note}
+This feature requires the `nextflow.preview.topic` feature flag to be enabled.
+:::
+
+The `topic` method is used to create a "topic" channel, which is a queue channel that can receive items from many sources implicitly based on a "topic" identifier.
```

Comment:
As mentioned in the main comments thread, I think that `topic` should be advertised as a new channel type, at the same level of `value` and `queue`. 

The distinctive trait is indeed the ability to collect input from multiple processes. (this is opposite to considering it a specialisation of the `queue` type)

Reply:
See discussion above